### ISSUE TITLE
list_jobs: Cannot mix placeholder styles "$1" and "?"

### ISSUE THREAD

**Issue by s1037989** [mojolicious/minion#87]
* Minion version: 9.13
* Perl version and Operating system:
```
$ mojo version
CORE
  Perl        (v5.22.1, linux)
  Mojolicious (8.23, Supervillain)

OPTIONAL
  Cpanel::JSON::XS 4.09+  (n/a)
  EV 4.0+                 (n/a)
  IO::Socket::Socks 0.64+ (0.74)
  IO::Socket::SSL 2.009+  (2.034)
  Net::DNS::Native 0.15+  (0.15)
  Role::Tiny 2.000001+    (2.000001)

This version is up to date, have fun!
```
* Other versions:
```
$ for i in Minion Mojo::Pg DBD::Pg; do echo -n "$i: "; perl -M$i -E "say $i->VERSION"; done
Minion: 9.13
Mojo::Pg: 4.15
DBD::Pg: 3.5.3
```

### Steps to reproduce the behavior
I browse to my_app/minion and it loads the dashboard fine.  I click on any of the four states such as "Finished" and I get a 500.  Locks and Workers is fine.  This started occurring in the past month or so.  Log:

```
[2019-09-04 01:14:43.63693] [9312] [error] Cannot mix placeholder styles "$1" and "?" at /usr/lib/x86_64-linux-gnu/perl5/5.22/DBD/Pg.pm line 277.
[2019-09-04 01:14:43.64418] [9312] [info] 1.2.3.4 GET https://my_app/minion/jobs?state=finished 500 0.0168s
```

### Expected behavior
`200`
Get a pretty web list of Finished jobs

### Actual behavior
`500` Rainbow-puking Unicorn Dino.
`Cannot mix placeholder styles "$1" and "?" at /usr/lib/x86_64-linux-gnu/perl5/5.22/DBD/Pg.pm line 277.`

**Comment by kraih** [comment]
This should be fixed in `DBD::Pg` 3.7.1.

**Comment by s1037989** [comment]
Success!  Thank you!
